src/app/views/forgotten-password/forgottenPasswordEmailSent.jsx

```diff
@@ -1,28 +1,41 @@
 import React from "react";
-import Input from "../../components/Input";
 import PropTypes from "prop-types";
+import Panel from "../../components/panel/Panel";
 
 const propTypes = {
     validationErrors: PropTypes.shape({
-        hading: PropTypes.string,
+        heading: PropTypes.string,
         body: PropTypes.oneOfType([PropTypes.string, PropTypes.element])
     }),
     onSubmit: PropTypes.func,
-    emailInput: PropTypes.shape({
-        id: PropTypes.string,
-        label: PropTypes.string,
-        type: PropTypes.string,
-        onChange: PropTypes.func,
-        error: PropTypes.string
-    })
+    email: PropTypes.string
 };
 
 const ForgottenPasswordEmailSent = props => {
-    // TODO this page is to be done in an upcoming task
+    const noEmailBody = (
+        <div className={"margin-top--1"}>
+            <p>
+                If you did not get the email you can
+                <a href={"/florence/forgotten-password"} className={"colour--primary-link"}>
+                    reset your password again.
+                </a>
+            </p>
+            <p>
+                Alternatively, to get help and support with your Florence account, please email&nbsp;
+                <a href="mailto:publishing@ons.gov.uk">publishing@ons.gov.uk</a>
+            </p>
+        </div>
+    );
     return (
         <div className="grid grid--justify-center">
-            <div className="grid__col-3">
+            <div className="grid__col-4">
                 <h1>We sent you an email</h1>
+                <p className={"font-size--18 margin-bottom--1"}>
+                    If you have a Florence account, we’ve sent an email to <b>{props.email}</b>.
```

Comment:
I think it is worth changing b to strong tag. 
```suggestion
                    If you have a Florence account, we’ve sent an email to <strong>{props.email}</strong>.
```